### DISCUSSION TITLE
Filter by organization in Project Beta

### DISCUSSION THREAD

**Discussion by michael-hawker** [community/community#19096] [Projects and Issues]
Looks like I can filter by repo in the Projects Table View, but not organization.

If I'm trying to add 'external' issues from other repos/orgs in my table, I have no way to just blanket say to exclude all my own organization issues. Instead, I have to filter out each repo within my org...

I'd really like to just be able to do `-organization:CommunityToolkit` or `-org:CommunityToolkit` and be able to only see the issues that come from other repos outside or organization (or vice-versa).

Related #4666

**Comment by rileybroughten** [comment]
Hi @michael-hawker, thank you for the feedback! We are always looking for ways to improve the filter experience and will take this back to the team! 👍

**Reply by michael-hawker** [reply]
Thanks @rileybroughten! If it helps this is what I'm trying to setup for us to track issues with our dependencies as a tab in our project:

![image](https://user-images.githubusercontent.com/24302614/175123266-b3be044d-caab-493a-9ba6-50ff21a5111c.png)

Also related to the [comment on the discussion here](https://github.com/github-community/community/discussions/5378#discussioncomment-2997230) around auto-adding issues to project boards based on some criteria.
